digitiser-aggregator/src/main.rs

```diff
@@ -199,8 +245,20 @@ async fn cache_poll(
                 .send(future_record, Timeout::After(Duration::from_millis(100)))
                 .await
             {
-                Ok(r) => debug!("Delivery: {:?}", r),
-                Err(e) => error!("Delivery failed: {:?}", e),
+                Ok(r) => {
+                    debug!("Delivery: {:?}", r);
+                    //  WARN: Check whether this should count as a message being processed.
+                    counter!(MESSAGES_PROCESSED).increment(1)
```

Comment:
No, this does not really make sense. There should be a metric that represents the number of complete frames sent by the aggregator.

In general, avoid trying to pigeon hole metrics into a few common names, create the metrics that make sense in a component first then combine the common ones where appropriate.